hpx/lcos/async.hpp

```diff
@@ -158,43 +193,23 @@ namespace hpx { namespace detail
         }
     };
 
-    ///////////////////////////////////////////////////////////////////////////
+    // BOOST_SCOPED_ENUM(launch)
     template <typename Policy>
     struct async_dispatch<Policy,
```

Comment:
This is the implementation of `async(launch::..., F, ...)`, i.e. for the local case. The remote case is implemented in `hpx/async.hpp`